tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_1.py

```diff
@@ -92,3 +92,36 @@ def test_raise_error_when_negative_f_cd_is_given(self) -> None:
                 diameter=diameter,
                 f_cd=f_cd,
             )
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (
+                    r"\Phi_{m,min} = \frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot \Phi} \right) }{f_{cd}} "
+                    r"= \frac{80.00 \cdot 1000 \cdot \left( \frac{1}{200.00} + \frac{1}{2 \cdot 16.00} \right)}{30.00} = 96.67"
```

Comment:
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/320bf846-6e82-481f-b964-f28afb067a57)
```suggestion
                    r"Ø_{m,min} = \frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot Ø} \right) }{f_{cd}} "
                    r"= \frac{80.00 \cdot 1000 \cdot \left( \frac{1}{200.00} + \frac{1}{2 \cdot 16.00} \right)}{30.00} = 96.67"
```